### PR TITLE
bpo-42151: don't set specified_attributes=1 in pure Python ElementTree

### PR DESCRIPTION
**Pull request by obfusk**
https://bugs.python.org/issue42151

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@obfusk

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by scoder** [comment]
Thanks for your contribution. The patch looks good to me.
Please add a News entry and send in a signed CLA form.

**Comment by obfusk** [comment]
> Please add a News entry

Done.

> and send in a signed CLA form.

Already on its way :)

**Comment by mattip** [comment]
Thanks @obfusk this solved the issue in PyPy.

File: Lib/test/test_xml_etree.py [hunk 1]
```diff
@@ -1354,6 +1367,12 @@ def test_tree_write_attribute_order(self):
         self.assertEqual(serialize(root, method='html'),
                 '<cirriculum status="public" company="example"></cirriculum>')
 
+    def test_attlist_default(self):
+        # Test default attribute values; See BPO 42151.
```
Comment (corona10) [inline Lib/test/test_xml_etree.py:1371]:
```suggestion
        # Test default attribute values; See bpo-42151.
```

File: Misc/NEWS.d/next/Library/2020-10-26-18-01-09.bpo-42151.et5f7s.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Make the pure Python implementation of `xml.etree.ElementTree` behave the
```
Comment (corona10) [inline Misc/NEWS.d/next/Library/2020-10-26-18-01-09.bpo-42151.et5f7s.rst]:
```
[2] ../Misc/NEWS.d/next/Library/2020-10-26-18-01-09.bpo-42151.et5f7s.rst:1: default role used
```

Documentation build is failed

**Review by corona10 — CHANGES_REQUESTED** [review]

**Comment by obfusk** [comment]
I have made the requested changes; please review again

**Review by corona10 — APPROVED** [review]

**Pull request merged**